packages/@aws-cdk/aws-apigatewayv2/test/http/stage.test.ts

```diff
@@ -1,8 +1,11 @@
 import { Template } from '@aws-cdk/assertions';
+import { Certificate } from '@aws-cdk/aws-certificatemanager';
 import { Metric } from '@aws-cdk/aws-cloudwatch';
 import { Stack } from '@aws-cdk/core';
-import { HttpApi, HttpStage } from '../../lib';
+import { DomainName, HttpApi, HttpStage } from '../../lib';
 
+const domainName = 'example.com';
+const certArn = 'arn:aws:acm:us-east-1:111111111111:certificate';
```

Comment:
define a `describe()` block around the new tests and scope these variables local to the block.

Reply:
Done!